### ISSUE TITLE
SyntaxVisualizerExtension is building with warnings

### ISSUE THREAD

**Issue by jaredpar** [dotnet/roslyn#17325]
Look at a any Microbuild build log and you will see the following:

> 01:29:51 C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\microbuilde0474b74\src\Tools\Source\SyntaxVisualizer\SyntaxVisualizerExtension\SyntaxVisualizerExtension.csproj]

This isn't being caught because Microbuild doesn't have the post processing step to fail builds on MSBuild warnings.  That's only done on the unit test legs.  Hence it's going unnoticed.  

The underlying issue is this particular reference in project.json:

``` json
"VSSDK.ComponentModelHost": "[12.0.4]"
```

The remainder of the project is referencing this DLL at version 14.0.0.0 but this brings it in at 12.0.0.0.  Hence the conflict occurs.

**Comment by Pilchie** [comment]
We don't support VS2013 anymore, so I would guess updating to 14.0 is fine.

**Comment by jaredpar** [comment]
Everything else in that project targets 14.0 so it should be fine.  

The problem is there is no actual 14.0 package for ComponentModelHost.  Need to create one for us to reference.

**Comment by AlexEyler** [comment]
I was just perusing the roslyn repo for VSSDK issues and saw this one that I can help out with! We uploaded ComponentModelHost for Dev14 a couple weeks ago: https://www.nuget.org/packages/Microsoft.VisualStudio.ComponentModelHost/14.0.25424.

**Comment by jaredpar** [comment]
@AlexEyler thanks! That let me fix the issue.
